### DISCUSSION TITLE
Add grid-based cursor movement.

### DISCUSSION THREAD

**Discussion by pit-ray** [pit-ray/win-vind#155] [Development]
If you want to participate in this developments, join this discussion.

The grid-based cursor movement has two layers (or three layers).
The first layer is based on #115.
The second layer is based on #92 or [warpd](https://github.com/rvaiya/warpd).

For more details, please read discussions of #145.

The topic branch of the first layer is [gridmove](https://github.com/pit-ray/win-vind/tree/gridmove).

**Comment by pit-ray** [comment]
I added the demo program of #115 into the latest commit in the gridmove branch.
<video src="https://user-images.githubusercontent.com/42631247/232825633-38a3ba1e-0087-4b34-8ecb-57b4d25e5eb5.mp4" controls></video>
I wish I had a better name for this function, but for now I am using gridmove.

If you try this demo, please type `<ctrl-m>` in GUI normal mode.
Its parameters are defined in [here](https://github.com/pit-ray/win-vind/blob/gridmove/src/core/settable.cpp#L12-L17).

For example, the following run-command changes its grid size.
```vim
set gridmove_l1size = 24x16
```

I would be glad to hear your opinions.

**Comment by pit-ray** [comment]
I released v5.1.0-beta1, including the grid-based movement (layer-1).
https://github.com/pit-ray/win-vind/releases/tag/v5.1.0-beta1

Please try it and give me your opnions.
Thanks.

**Comment by imshenwei** [comment]
great!!!!!! Thanks for your work!!!!! i would defenitely try it later this or next week!!! ON MOVE!!!!

**Comment by pit-ray** [comment]
[v5.1.0](https://github.com/pit-ray/win-vind/releases/tag/v5.1.0) is available now.
